src/drivers/uavcan/uavcan_params.c

```diff
@@ -292,7 +316,7 @@ PARAM_DEFINE_INT32(UAVCAN_SUB_FLOW, 0);
  * @reboot_required true
  * @group UAVCAN
  */
-PARAM_DEFINE_INT32(UAVCAN_SUB_GPS, 1);
+PARAM_DEFINE_INT32(UAVCAN_SUB_GPS, 0);
```

Comment:
Maybe we just set these defaults for saluki board specific files instead?

Reply:
yep, I moved it there https://github.com/tiiuae/saluki-v1/pull/69

Reply:
great thanks :)